tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -39,27 +39,27 @@ Define {
 #
 #ALH Index: 0, Direction: duplex
 Object.Dai {
-	ALH."2" {
+	ALH.0 {
```

Comment:
not completely following the formula. 

a) what is the relationship between pcm_id and ALH? there could be M:1 and 1:N topologies.
b) what is a route_id
c) why does the pipeline come into play.

That was a long day, bear with me

Reply:
> not completely following the formula.

@plbossart Please see https://github.com/thesofproject/sof/pull/6549/commits/f554a2f1714bec1c3b8b76844faebba8e5788096 for your reference. I am trying to follow the same customary to assign ALH index. 

> 
> a) what is the relationship between pcm_id and ALH? there could be M:1 and 1:N topologies. 

ALH index = pcm_id * 10 + N, where N could be 0 to 9.
For example, say there is an aggregated speaker device
pcm_id = 2, and the ALH index will be 20, 21.

> b) what is a route_id 

For example,
route.**0** {
		source	"gain.1.1"
		sink	"copier.ALH.1.1"
	}
The route_id above is 0

> c) why does the pipeline come into play.

The pipeline id also follows the same rule.


> 
> That was a long day, bear with me

Reply:
@plbossart Are you good with this PR?